### ISSUE TITLE
Support for multiple Users/Orgs 

### ISSUE THREAD

**Issue by CrazyWolf13** [jaedle/mirror-to-gitea#25]
Hi

Not sure whether this is already possible or not, but I couldn't find it in the main readme.

It would be really awesome, if it would be possible to mirror not just one org/user per docker container, instead multiple users and/or orgs.

Btw I really like this tool!

**Comment by jaedle** [comment]
Hey 👋 

Thanks for opening the issue.

Right now it is only possible per one container instance to mirror a single org / user to Gitea.

Could you please elaborate your use-case a bit in depth?

Best,
Dennis

**Comment by CrazyWolf13** [comment]
Hi
Thanks for the insanely quick reply!

Well as I'd like to manage my own mirror, mirroring multiple production neccessary repositorys, is crucial for me.

For me this includes a lot of different repository and users, as I'm very active in Homelabbing and selfhosting.

Having to use a single docker container for each org/ user with multiple repos is just an insane overhead.

Thanks!

**Comment by outbackdingo** [comment]
I tend to agree, i use it to mirror my personal things, but i also am part of 3 organizations that require mirroring. a bit hard to do this way

**Comment by jaedle** [comment]
Hey 👋 

I guess that's a viable use-case. I will speed up the refactoring + adding tests and afterwards release a stable version (probably 1.0) which will support a configuration file including those requirements.

**Comment by jaedle** [comment]
Done by @arunavo4
